draft-ietf-tls-esni.md

```diff
@@ -85,10 +85,12 @@ both.
 The protocol specified by this document takes a different approach. It assumes
 that private origins will co-locate with or hide behind a provider (reverse
 proxy, application server, etc.) that protects SNIs for all of the domains it
-hosts. As a result, SNI protection does not indicate that the client is
-attempting to reach a private origin, but only that it is going to a particular
-service provider, which the observer could already tell from the visible IP
-address.
+hosts. These co-located servers form an anonymity set wherein all elements
+have a consistent configuration, e.g., set of supported application protocols,
+ciphersuites, TLS versions, and so on. SNI protection reveals that a client is
+connecting to a particular service provider without revealing the specific
```

Comment:
"a particular service provider without revealing the specific private serer from the anonymity set" -> "a particular service provider, but does not reveal which server terminates the connection".